src/main/java/uk/gov/justice/hmpps/casenotes/controllers/CaseNoteController.java

```diff
@@ -163,4 +167,39 @@ public CaseNoteType updateCaseNoteSubType(
             @RequestBody @NotNull final UpdateCaseNoteType body) {
         return caseNoteService.updateCaseNoteSubType(parentType, subType, body);
     }
+
+    @GetMapping("/events_no_limit")
+    @ApiIgnore
+    @ApiOperation(value = "Fetch all case notes of a given type since a given date", notes = "Returns all case notes - consumed by Delius<br/>" +
+            "These are generated by  whenever a case note is created or amended for an offender.<br/>" +
+            "*Note:* An alternative call [GET /case_notes/for_delius](#case-notes-feed-fetch-case-notes-direct-get) has been created for performance reasons.<br/>" +
+            "Some case notes are automatically generated by the system in response to an action relating to the offender e.g. An alert being raised or a prison transfer.<br/>" +
+            "The note type only filters at the top note type level not the sub type.<br/>" +
+            "note_type can be presented multiples times in the URL to filter by multiple note types.", nickname = "getCaseNotesEvents")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "OK", response = CaseNoteEvent.class, responseContainer = "List"),
```

Comment:
I think you can remove the resposneContainer and replace response= CaseNoteEvent.class with reponse = CaseNoteEvents.class